libraries/image/src/custom-mobilenet.ts

```diff
@@ -93,6 +95,8 @@ const isAlphaValid = (version: number, alpha: number) => {
 const parseModelOptions = (options?: ModelOptions) => {
     options = options || {};
 
+    if (options.imageSize) IMAGE_SIZE = options.imageSize;
```

Comment:
Create a new local variable here that defaults to DEFAULT_IMAGE_SIZE

Reply:
Forgive me if I'm misunderstanding, but I don't think a local is going to work here. The point of setting it globally is so the predict functions can use it in the custom-mobilenet class